homeassistant/components/adaptive_lighting/const.py

```diff
@@ -0,0 +1,141 @@
+"""Constants for the Adaptive Lighting integration."""
+import voluptuous as vol
+
+from homeassistant.components.light import VALID_TRANSITION
+import homeassistant.helpers.config_validation as cv
+
+ICON = "mdi:theme-light-dark"
+
+DOMAIN = "adaptive_lighting"
+SUN_EVENT_NOON = "solar_noon"
+SUN_EVENT_MIDNIGHT = "solar_midnight"
+
+CONF_NAME, DEFAULT_NAME = "name", "default"
+CONF_LIGHTS, DEFAULT_LIGHTS = "lights", []
+CONF_DISABLE_BRIGHTNESS_ADJUST, DEFAULT_DISABLE_BRIGHTNESS_ADJUST = (
```

Comment:
Thanks for your work on this, Bas! 

On these 3 configuration options (disable_brightness_adjust, disable_color_temp_adjust, disable_rgb_color_adjust), I’d suggest to avoid the double negative of disable = false as is common in many other integrations and rather use adapt_brightness, adapt_color_temp, adapt_rgb_color (then it’s also consistent with the integration name) with a default of True.

Reply:
Great suggestion! I will implement it ASAP.

Reply:
I renamed to adjust_brightness, adjust_color_temp, adjust_rgb_color.

Thanks for the suggestion :tada: